webapp/components/DeleteAccount/DeleteAccount.spec.js

```diff
@@ -0,0 +1,168 @@
+import { config, mount, createLocalVue } from '@vue/test-utils'
+import DeleteAccount from './DeleteAccount.vue'
+import Styleguide from '@human-connection/styleguide'
+import Vuex from 'vuex'
+
+const localVue = createLocalVue()
+
+localVue.use(Vuex)
+localVue.use(Styleguide)
+config.stubs['b-switch'] = '<span><slot /></span>'
+config.stubs['b-checkbox'] = '<span><slot /></span>'
+
+describe('DeleteAccount.vue', () => {
+  let mocks
+  let wrapper
+  let getters
+  let actions
+  let deleteAccountBtn
+
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(),
+      $apollo: {
+        mutate: jest
+          .fn()
+          .mockResolvedValueOnce({
+            data: {
+              DeleteAccount: {
+                id: 'u343',
+              },
+            },
+          })
+          .mockRejectedValue({ message: 'Not authorised!' }),
+      },
+      $toast: {
+        error: jest.fn(),
+        success: jest.fn(),
+      },
+      $router: {
+        history: {
+          push: jest.fn(),
+        },
+      },
+    }
+    getters = {
+      'auth/user': () => {
+        return { id: 'u343' }
+      },
+    }
+    actions = { 'auth/logout': jest.fn() }
+  })
+
+  describe('mount', () => {
+    const Wrapper = () => {
+      const store = new Vuex.Store({
+        getters,
+        actions,
+      })
+      return mount(DeleteAccount, { mocks, localVue, store })
+    }
+
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+
+    afterEach(() => {
+      jest.clearAllMocks()
+    })
+
+    it('defaults to deleteContributions to false', () => {
+      expect(wrapper.vm.deleteContributions).toEqual(false)
```

Comment:
I think since the bug in #785 I'm reluctant to `wrapper.vm`. Couldn't we refactor the test here to actually do sth. with the component or see sth. in the html to check it?

Reply:
I tried to click on the switch, and on the check boxes, but they aren't available... at least that I could find